### ISSUE TITLE
Numericality fails to validate decimal number starting with a dot

### ISSUE THREAD

**Issue by karellm** [DockYard/ember-validations#321]
In javscript (and in real life), `.2` is considered a number and is equivalent to `0.2`. 

The entire `numericality` validator fails though if you input `.2`. To check numericality, you could simply do:

```javascript
import Base from 'ember-validations/validators/base';

export default Base.extend({
  call() {
    let number = this.model.get(this.property);
    if (isNaN(number)) {
      this.errors.pushObject('not a number');
    }
  }
});
```

Is it an intended behaviour? Can you fix it or would you accept a patch?

**Comment by Truffula** [comment]
@karellm, the base numericality pattern is more forgiving than isNaN (with the exception of this issue) in that it also allows for values like "5,000,000.25" - although you can't use `parseFloat` on that number without removing the commas first.

The existing check (in patterns.js) is:
```js
/^(-|\+)?(?:\d+|\d{1,3}(?:,\d{3})+)(?:\.\d*)?$/
```
I think this captures your use case:
```js
/^(-|\+)?(?:(?:(?:\d+|\d{1,3}(?:,\d{3})+)(?:\.\d*)?)|(?:\.\d+))$/
```

**Comment by karellm** [comment]
I ran a quick test and it seems to work perfectly! Are you taking care of fixing it or do you want a PR?

**Comment by Truffula** [comment]
@karellm PR done! :)

**Comment by karellm** [comment]
I couldn't find the PR. Are you a collaborator on the project or just helping out?

**Comment by Truffula** [comment]
Sorry, I missed a step. Just helping out! :)

**Comment by martndemus** [comment]
I am going to close this issue for now, feel free to reopen this issue if you feel like it's still relevant and we should take a look at it.

**Comment by karellm** [comment]
@martndemus It definitely is relevant though non blocking

**Comment by martndemus** [comment]
@karellm Fixed by #424
